frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -60,6 +69,28 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
   const replica = isArbiterEnabled ? OCS_DEVICE_SET_ARBITER_REPLICA : OCS_DEVICE_SET_REPLICA;
   const name = getName(ocsConfig);
 
+  React.useEffect(() => {
+    if (selectedSCName) {
+      if (isArbiterEnabled && !isArbiterSC(null, pvData, nodesData, selectedSCName)) {
+        setError(
+          t(
+            'ceph-storage-plugin~The Arbiter stretch cluster requires a minimum of 4 nodes (2 different zones, 2 nodes per zone). Please choose a different storage class or create a new volume set that matches the minimum node requirement',
+          ),
+        );
+      } else if (
+        !hasFlexibleScaling &&
+        !isArbiterEnabled &&
+        !isNormalSC(null, pvData, nodesData, selectedSCName)
```

Comment:
```suggestion
        !hasFlexibleScaling &&
        !isNormalSC(null, pvData, nodesData, selectedSCName)
```

Reply:
this will cause a bug... if `isArbiterEnabled === true && !isArbiterSC === false` then control will fall to L81 and here `!hasFlexibleScaling === true && !isNormalSC === true` (beacuse current storage-class has nodes w.r.t to arbiter requirements, so it is not normal)... so it will start showing the incorrect error: `"The OCS Storage cluster requires a minimum of 3 node....."` even though my nodes/pvs configurations are correct w.r.t arbiter mode...

Reply:
I would like to see it made simpler - 
if (isArbiterEnabled) return isArbiterSC();
if (!isFlexibleScaling) return isInternalAttachedSC()